theforeman.org/scripts/test/test_hammer_cli_foreman_discovery.sh

```diff
@@ -6,7 +6,6 @@
 gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
 rvm use ruby-${ruby}@${gemset} --create
 rvm gemset empty --force
-gem update --no-document
 gem install bundler -v '< 2.0' --no-document
```

Comment:
I wonder if by now we can't drop this as well.

Reply:
does rvm install bundler by default now? i imagine we don't need to specify '< 2.0' anymore even if it doesn't